### PR TITLE
Add initial build & dev doc for linux

### PR DESCRIPTION
**Pull request by Ozaq**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,113 @@
...
+* wget
+
+* gcc or clang (c++17 support is required)
+
+Additionally you will need a few python packages.It is recomended to install all
+dependencies listed in `requirements.txt` into a virtual environment, e.g.:
+
+```bash
+python -m venv ~/simulator-venv
+source ~/simulator-venv
```
Comment (schroedtert) [inline README.md:33]: Shouldn't this be: `source ~/simulator-venv/bin/activate`

File: README.md [hunk 2]
```diff
@@ -1 +1,113 @@
...
+the sources:
+
+```bash
+mkdir ~/simulator-build
+cd ~/simulator-build
+cmake -DCMAKE_PREFIX_PATH=~/simulator-deps -DCMAKE_BUILD_TYPE=Debug <path-to-repository>/cpp
+cmake --build . -j
+```
+
+Above command assumes that third party dependencies that your build with
```
Comment (schroedtert) [inline README.md:59]:
```suggestion
Above command assumes that third party dependencies that you build with
```

File: README.md (hunk 1 above)
Comment (Ozaq) [reply README.md]: should be done now

**Review by zeroset — APPROVED** [review]
I missed to review this, sorry. Very nice. Thanks

**Pull request merged**